notebooks/238-deepfloyd-if/238-deep-floyd-if-convert.ipynb

```diff
@@ -103,7 +103,7 @@
     "\n",
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #5.    %pip install -q "openvino-nightly" opencv-python
</pre><p>openvino-nightly may have conflicts with existing environment, we recommend to make sure that you have not installed other libs with ov before installing it</p><p> </p><p>%pip uninstall -q -y openvino-dev openvino openvino-nightly</p><p>%pip install -q openvino-nightly</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/1602/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>

Reply:
Added